lib/build.js

```diff
@@ -592,7 +596,7 @@ const buildAct = config => ({act, oog, pass, name, hasGas, gas, lemma}) => {
       balance: balance,
       code:    alias + "_bin_runtime",
       storage: storage,
-      origStorage: "_",
+      origStorage: origStorage,
```

Comment:
Here is where we didn't use to make assumptions about storage that are relevant to gas at the beginning of the call.